config/coaddBase.py

```diff
@@ -1,4 +1,2 @@
 # Configs shared between makeWarp and assemble
 config.matchingKernelSize = 29
-config.doApplyExternalPhotoCalib = True
-config.doApplyExternalSkyWcs = True
```

Comment:
By removing these `doApply...` configs (here and elsewhere), are we also removing the flexibility to turn these components off? I see a new `useVisitSummaryPsf` config option - do we also need similar `useVisitSummaryPhotoCalib` and `useVisitSummaryWcs` configs to replicate this functionality?

Reply:
No, we are intentionally dropping this functionality _here_, because the expectation is that controlling what goes into a visit summary dataset (which still has these knobs) is sufficient for everything downstream of that visit summary and upstream of any other visit summary.

Reply:
Got it, thanks for the explanation, good to know!